src/map/packets.hpp

```diff
@@ -914,6 +914,31 @@ struct PACKET_ZC_NOTIFY_EFFECT {
 } __attribute__((packed));
 DEFINE_PACKET_HEADER(ZC_NOTIFY_EFFECT, 0x19b);
 
+#if PACKETVER >= 20120618
+struct PACKET_ZC_EFST_SET_ENTER{
+	uint16 packetType;
+	uint32 tid;
+	uint16 type;
+	uint32 duration;
+	uint32 duration2;
+	uint32 val1;
+	uint32 val2;
+	uint32 val3;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_EFST_SET_ENTER, 0x984)
+#elif PACKETVER >= 20111108
+struct PACKET_ZC_EFST_SET_ENTER{
+	uint16 packetType;
+	uint32 tid;
+	uint16 type;
+	uint32 duration;
+	uint32 val1;
+	uint32 val2;
+	uint32 val3;
```

Comment:
For me valx should not be unsigned even though I don't remember a case where it is negative in our sc

Reply:
i'm in change of this now because i asked same to myself 😄 but I was going to ask for confirmation here